models/forecast_currently_object.js

```diff
@@ -0,0 +1,8 @@
+class ForecastCurrentlyObject {
+  constructor(location, forecast){
+    this.location = location
+    this.currently = forecast.currently
+  }
+}
+
+module.exports = ForecastCurrentlyObject
```

Comment:
Thank you. After further discussing this with you,  we decided this should be left in the model folders since this object is being passed to a presenter.